app/concepts/cell/documentation/documentation.erb

```diff
@@ -1,46 +1,39 @@
 <section>
-  <div class="lg:hidden bg-bg-blue text-white fixed left-0 top-20 pr-1 py-3 rounded-r" id="sideNavShowButton" style="writing-mode: vertical-rl; text-orientation: upright;">
+  <div class="fixed left-0 py-3 pr-1 text-white rounded-r lg:hidden bg-bg-blue top-20" id="sideNavShowButton" style="writing-mode: vertical-rl; text-orientation: upright;">
     Chapters
   </div>
   <div class="lg:flex">
     <nav class="documentation-side-nav" id="sideNav">
-      <div class="lg:hidden absolute right-4 top-4 text-3xl text-white" id="sideNavHideButton">
+      <div class="absolute text-3xl text-white lg:hidden right-4 top-4" id="sideNavHideButton">
         X
       </div>
       <%= toc_left %>
     </nav>
-    <div class="lg:p-10 pl-8 pr-4 py-10 bg-light-grey grow">
-      <h1 class="lg:text-3xl text-2xl text-blue font-bold">
-        <span class="font-black uppercase">
-          <%= documentation_title %>
-        </span>
-        Documentation
-      </h1>
-      <!-- <div class="text-base leading-10">
-        <span class="font-bold text-bg-blue">
-          <%= "Subject A |" %>
-        </span>
-        <span class="text-purple">
-          Subject 02
-        </span>
-      </div> -->
-      <div class="xl:flex xl:gap-0.5 mt-5">
-        <div class="max-w-[57rem] lg:p-8 lg:pb-14 p-4 bg-white text-bg-blue">
+    <div class="py-10 pl-8 pr-4 bg-white lg:p-10 grow">
+      <div class="xl:flex xl:gap-0.5">
+        <div class="w-full p-4 lg:max-w-3xl lg:p-8 lg:pb-14 text-bg-blue xl:border-r-2 xl:border-light-grey" id="documentation">
+          <h1 class="pb-6 text-2xl font-bold border-b-2 lg:text-3xl text-blue">
+            <span class="font-black">
+              <%= documentation_title %>
+            </span>
+            Documentation
+          </h1>
+
+          <!-- <div class="text-base leading-10">
+            <span class="font-bold text-bg-blue">
+              <%= "Subject A |" %>
+            </span>
+            <span class="text-purple">
+              Subject 02
+            </span>
+          </div> -->
+
           <%= yield %>
         </div>
-        <%= tocs_right %>
+        <div class="relative" id="right-toc">
+          <%= tocs_right %>
+        </div>
       </div>
     </div>
   </div>
 </section>
-<section class="px-10 py-22 bg-bg-purple-3">
```

Comment:
@apotonick I removed the call out section at the bottom as it was causing some UI conflicts. IMO, it seems redundant given we've same button in top nav on the screen. Do you think we should keep it ?

<img width="1440" alt="Screenshot 2023-09-02 at 10 12 27 PM" src="https://github.com/apotonick/trailblazer-front/assets/6301525/70a154bf-9255-4fb1-8d6a-02ebe58ca103">

Reply:
We should keep that and rather fix the menu to not slide into it?

Reply:
Sure, I've added this section back.